firmware/common/i2cdev_ppmod.hpp

```diff
@@ -45,6 +49,20 @@ class I2cDev_PPmod : public I2cDev {
 
         // will respond with application data
         COMMAND_APP_TRANSFER = 0x4183,
+
+        // UART specific commands
+        COMMAND_UART_REQUESTDATA_SHORT = USER_COMMANDS_START,
```

Comment:
hmmm
I intended USER_COMMANDS_START for a place where the "per module" api is placed that is not "known" by the firmware. It can be used by the standalone app to communicate with its module.
therefore I did not intend to have dependencies in our firmware to other modules.

especially the PP MDK, because otheres will use it to build new modules and should not have the impression they need to change stuff in the main firmware.

Reply:
shared things like GPS that are directly supported by the firmware however should be here. just don't put anything after USER_COMMANDS_START